WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Notes Section/Customer Note/EditCustomerNote.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import SwiftUI
+
+/// Hosting controller that wraps an `EditCustomerNote` view.
+///
+final class EditCustomerNoteHostingController: UIHostingController<EditCustomerNote> {
+    init() {
+        super.init(rootView: EditCustomerNote())
+
+        // Needed because a `SwiftUI` cannot be dismissed when being presented by a UIHostingController
+        rootView.dismiss = { [weak self] in
+            self?.dismiss(animated: true, completion: nil)
+        }
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Allows merchant to edit the customer provided note of an order.
+///
+struct EditCustomerNote: View {
+
+    /// Set this closure with UIKit dismiss code. Needed because we need access to the UIHostingController `dismiss` method.
+    ///
+    var dismiss: (() -> Void) = {}
+
+    // TODO: Replace with view model backed value
+    @State private var textContent = "Tap and edit me"
+
+    var body: some View {
+        NavigationView {
+            TextEditor(text: $textContent)
+                .padding()
+                .navigationTitle(Localization.title)
+                .navigationBarTitleDisplayMode(.inline)
+                .navigationBarItems(trailing: Button(Localization.done) { // I couldn't find a way to make the "Done" button bold using a toolbar :-(
```

Comment:
But it looks bold to me, just like in Figma design. Am I missing something?
<img width=400 src="https://user-images.githubusercontent.com/3132438/130216211-625f1eb0-8f66-4087-81b8-47d1d88e9629.png">

Reply:
Apple recommends using `.toolbar { }` when placing buttons on a navigation bar, as it will work better with `watchOS` and `tvOS`. However, using `.toolbar { }` makes the button to be non-bold and I couldn't find a way to style the button. 

Using `.navigationBarItems(trailing:` does makes the button bold by default. I added that comment to let readers know why the cancel button is within a `.toolbar { }` and why the done button is not.

Reply:
Got it, thanks for clarification!